modules/howtos/examples/subdoc.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const couchbase = require('couchbase')
+const { StoreSemantics } = couchbase
```

Comment:
not sure this needs a named import, where everything else indexes via `couchbase.` ?

Reply:
Ah yeah, I think that happened automagically and I didn't notice, 👍 will keep consistent with everything else.